internal/console/version.go

```diff
@@ -0,0 +1,20 @@
+package console
+
+import (
+	"fmt"
+
+	"github.com/spf13/cobra"
+)
+
+var versionCmd = &cobra.Command{
+	Use:   "version",
+	Short: "Shows the kics's current version",
+	RunE: func(cmd *cobra.Command, args []string) error {
+		fmt.Printf("%s\n", getVersion())
```

Comment:
@felipe-avelar @ruigomescx 

What do you guys think if we also print the current OPA lib version:

```
import ( 
 opaVersion "github.com/open-policy-agent/opa/version"
)

fmt.Printf("%s\n", getVersion())
fmt.Printf("OPA version: %s\n", opaVersion.Version)
```

Reply:
right now let's only print the kics version